packages/core/src/components/portal/portal.tsx

```diff
@@ -21,6 +21,11 @@ export interface IPortalProps extends IProps {
      * Callback invoked when the children of this `Portal` have been added to the DOM.
      */
     onChildrenMount?: () => void;
+
+    /**
+     * The HTML element that children will be mounted to. Default value is `document.body`.
```

Comment:
```suggestion
     * The HTML element that children will be mounted to. 
     * @default document.body
```
just like you did in all the other files